<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lab6</class>
 <widget class="QWidget" name="Lab6">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>321</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btn_error">
        <property name="text">
         <string>Error</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_warning">
        <property name="text">
         <string>Warning</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_information">
        <property name="text">
         <string>Information</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="btn_close">
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>270</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ИВТ-212, Чередников Максим</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
